packages/core/src/components/alert/alert.tsx

```diff
@@ -4,7 +4,7 @@
  * Licensed under the terms of the LICENSE file distributed with this project.
  */
 
-import * as classNames from "classnames";
+import classNames from "classnames";
 import * as React from "react";
```

Comment:
don't we need to do the same for all `import *` statements? I believe namespace imports are now discouraged altogether, so we should make that change across the board in this PR.

Reply:
This only needed for things that are effectively “module.exports = something” and don’t have flags marking them as originally es modules. 

TS has been doing synthetic wrong. Now you’re doing it right! (Namespaces are not invocable)

Reply:
seems the only offenders were `classnames`, `moment`, `moment-timezone`

Reply:
cool, so we're still blocking this PR on fixing `import * as moment` right?

Reply:
no i did that